.github/actions/conditional/conditions

```diff
@@ -33,6 +33,8 @@ docs/documentation/                         documentation
 
 js/                                         js
 rest/admin-ui-ext/                          js
+js/apps/account-ui/                         ci-webauthn
+js/libs/ui-shared/                          ci-webauthn
```

Comment:
This is missing the option `ci`, and without that change the build in `ci.yml` will not run.

```suggestion
js/apps/account-ui/                         ci ci-webauthn
js/libs/ui-shared/                          ci ci-webauthn
```

Reply:
> Unfortunately the test runs you referenced are a bit off as those trigger other areas due to your code changes around the actions and conditions.

Ahhh, you're right.

I'd like to prevent executing the whole CI when only some changes are made in the JS modules - only the WebAuthn? But as I see it now, it'd require some more changes in the ci.yml, and I'm not sure if it's worth worsening the ci.yml maintainability. I'll try to think it through a little bit more.

Reply:
The `needs.conditional.outputs.ci == 'true'` is specific to the build only, so I don't see a problem here. There are then a lot of default, but those should be rather fast. 

With the kind of problem it tries to prevent (which is currently bothering us quite a lot), I'd start with something that works first, and then refactor.